### ISSUE TITLE
Great stuff!  Some requests...

### ISSUE THREAD

**Issue by imkane** [FbF/Laravel-Blog#3]
I love how easy it is to integrate into an existing Laravel install, and the simplicity of its use.

Do you have any plans to add Tags, Categories, or the ability to Preview Draft posts?   Those are a few things I think that would really round out its functionality.

**Comment by neilcrookes** [comment]
I'm sure I will have a work project that needs that functionality at some point, but I haven't yet. When I do though, I'll add that functionality unless someone else adds them first ;-)

**Comment by imkane** [comment]
I wish I was a better programmer, then I could work on it lol

**Comment by MECU** [comment]
I would really like tags or categories as well. An ability to search/display posts by tags or categories would be awesome as well.

**Comment by neilcrookes** [comment]
I've provided the ability to extend the Post model in your app and attach posts to any other model and then filter by that model. It's really flexible and will allow you to attach posts to categories or tags via belongsTo or belongsToMany relationships. I've also release a new package to allow you to build categories, and in the readme I've provided step by step instructions on how to integrate the two packages. See release v0.5.0.
